<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamClass</class>
 <widget class="QMainWindow" name="ExamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="categoryListWidget"/>
      </item>
      <item>
       <widget class="QListWidget" name="shoppingListWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Category:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="categoryLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="categoryPushButton">
      <property name="text">
       <string>Show items</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Exam.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>categoryPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExamClass</receiver>
   <slot>showItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showItems()</slot>
 </slots>
</ui>
